pkg/url/url.go

```diff
@@ -981,7 +1058,11 @@ func Push(client *occlient.Client, kClient *kclient.Client, parameters PushParam
 			if urlInfo.Spec.Kind == envinfo.INGRESS && kClient == nil {
 				continue
 			}
-
+			// if parameters.IsExperimentalModeEnabled && kClient != nil {
+			// 	// route/ingress name is defined as <urlName>-<componentName>
+			// 	// to avoid error due to deplicate ingress name defined in different devfile components
```

Comment:
`duplicate`?

Reply:
These were comments I forgot to delete. Removed.